src/components/Calculator.js

```diff
@@ -1,39 +1,58 @@
+        <button type="button" className="button" onClick={() => handleClick('+/-')}>+/-</button>
+        <button type="button" className="button" onClick={() => handleClick('%')}>%</button>
+        <button type="button" className="button operation" onClick={() => handleClick('÷')}>÷</button>
       </div>
       <div className="column">
-        <input type="button" value="7" className="button" />
-        <input type="button" value="8" className="button" />
-        <input type="button" value="9" className="button" />
-        <input type="button" value="x" className="button operation" />
+        <button type="button" className="button" onClick={() => handleClick('7')}>7</button>
+        <button type="button" className="button" onClick={() => handleClick('8')}>8</button>
+        <button type="button" className="button" onClick={() => handleClick('9')}>9</button>
+        <button type="button" className="button operation" onClick={() => handleClick('x')}>x</button>
       </div>
       <div className="column">
-        <input type="button" value="4" className="button" />
-        <input type="button" value="5" className="button" />
-        <input type="button" value="6" className="button" />
-        <input type="button" value="-" className="button operation" />
+        <button type="button" className="button" onClick={() => handleClick('4')}>4</button>
+        <button type="button" className="button" onClick={() => handleClick('5')}>5</button>
+        <button type="button" className="button" onClick={() => handleClick('6')}>6</button>
+        <button type="button" className="button operation" onClick={() => handleClick('-')}>-</button>
       </div>
       <div className="column">
-        <input type="button" value="1" className="button" />
-        <input type="button" value="2" className="button" />
-        <input type="button" value="3" className="button" />
-        <input type="button" value="+" className="button operation" />
+        <button type="button" className="button" onClick={() => handleClick('1')}>1</button>
+        <button type="button" className="button" onClick={() => handleClick('2')}>2</button>
+        <button type="button" className="button" onClick={() => handleClick('3')}>3</button>
+        <button type="button" className="button operation" onClick={() => handleClick('+')}>+</button>
       </div>
       <div className="column">
-        <input type="button" value="0" className="button zero" />
-        <input type="button" value="." className="button" />
-        <input type="button" value="=" className="button operation" />
+        <button type="button" className="button zero" onClick={() => handleClick('0')}>0</button>
+        <button type="button" className="button decimal" onClick={() => handleClick('.')}>.</button>
+        <button type="button" className="button operation" onClick={handleEqual}>=</button>
       </div>
     </div>
   );
 }
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/235282695-7b7019e0-990e-49d4-a975-bc335477d5f6.png)

- [ ] As you can see in the image above that in the project requirements, you are supposed to practice state and props 😊 and you are practicing state but not props 😨 Kindly make a Button component for all the buttons in this calculator app and import that here in this Calculator.js and use props to send data to the Button.js component 👍

Reply:
I've created the Button component and also used props to send data to the Button.js